travis/before-deploy.sh

```diff
@@ -0,0 +1,5 @@
+if [[ "$TRAVIS_TAG" = "" ]]; then exit 0; fi
+
+PLATFORM=$([ "$TRAVIS_OS_NAME" == "linux" ] && echo "linux64" || echo "$TRAVIS_OS_NAME")
```

Comment:
I think we'd want to go by the `npm` conventions if possible:

`os`: `darwin | linux | win32`
`cpu`: `ia32 | x64 | arm | mips`

At least for the ones we will actually support.